idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/annotators/coercion/coercionAnnotator.kt

```diff
@@ -0,0 +1,12 @@
+package arrow.meta.ide.plugins.proofs.annotators.coercion
+
+import arrow.meta.ide.IdeMetaPlugin
+import arrow.meta.phases.Composite
+import arrow.meta.phases.ExtensionPhase
+
+val IdeMetaPlugin.coercionAnnotator: ExtensionPhase
+  get() = Composite(
+    coercionAnnotatorImplicit,
+    coercionAnnotatorExplicitProperty,
+    coercionAnnotatorExplicitValArgument
```

Comment:
```suggestion
    implicitCoercion,
    explicitPropertyCoercion,
    explicitValArgumentCoercion
```
Do you think that reads better. @danimontoya ?

Reply:
yes! will change it 👌